dask/dataframe/io/parquet/core.py

```diff
@@ -268,15 +307,24 @@ def read_parquet(
     if index and isinstance(index, str):
         index = [index]
 
+    if chunksize or (
+        split_row_groups and int(split_row_groups) > 1 and aggregate_files
+    ):
+        # Require `gather_statistics=True` if `chunksize` is used,
+        # or if `split_row_groups>1` and we are aggregating files.
+        gather_statistics = True
```

Comment:
I'm wondering if this justifies a log/warning. Maybe even raise if the user input is `False`?

Reply:
I agree - If the user has specified `False` we need to raise an error.